src/Entities/BadelineBoosts/BadelineBoostKeepHoldables.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Mono.Cecil.Cil;
+using MonoMod.Cil;
+using MonoMod.RuntimeDetour;
+using MonoMod.Utils;
+using System.Reflection;
+
+namespace Celeste.Mod.CommunalHelper.Entities.BadelineBoosters;
```

Comment:
I'm not sure whether a new namespace + sub-directories (both in C# project and Lönn plugins) were necessary. Also, we usually limit namespaces to `Celeste.Mod.CommunalHelper.Entities`. Yes there are some entities for which we forgot to do this, and we can't change it back because it may break maps

Reply:
The extra `BadelineBoosters` namespace is an oversight on my part, oops. I can remove that once I have access to VS again.

Reply:
NP; that'd be great, thanks!